<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>accountant</class>
 <widget class="QWidget" name="accountant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>账户表</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="maximumSize">
      <size>
       <width>360</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>开户</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="10" column="2" colspan="5">
        <widget class="QPushButton" name="AccClear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="12" column="2" colspan="5">
        <widget class="QPushButton" name="Confirm">
         <property name="text">
          <string>Confirm</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="3" colspan="4">
        <widget class="QLineEdit" name="AccExtra">
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>透支额</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QCheckBox" name="isChecking">
         <property name="text">
          <string>支票账户</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="3" colspan="3" alignment="Qt::AlignHCenter">
        <widget class="QCheckBox" name="isSaving">
         <property name="text">
          <string>储蓄账户</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>利率</string>
         </property>
        </widget>
       </item>
       <item row="8" column="6">
        <widget class="QComboBox" name="currencyType">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::InsertAtBottom</enum>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>人民币</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>美元</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>欧元</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>日元</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>港币</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="3" colspan="2">
        <widget class="QLineEdit" name="AccRate">
         <property name="text">
          <string>777</string>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>余额</string>
         </property>
        </widget>
       </item>
       <item row="8" column="5" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>币种</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>账户类型</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3" colspan="4">
        <widget class="QLineEdit" name="AccBank"/>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>账户号</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>开户支行</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3" colspan="4">
        <widget class="QLineEdit" name="AccID"/>
       </item>
       <item row="5" column="3" colspan="2">
        <widget class="QLineEdit" name="AccPoss"/>
       </item>
       <item row="5" column="5">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>开户日期</string>
         </property>
        </widget>
       </item>
       <item row="5" column="6">
        <widget class="QDateEdit" name="dateEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
